docs/add_a_validation.md

```diff
@@ -0,0 +1,317 @@
+    {
+        $fieldDefinitionForm
+            ->add('authorList', TextType::class, [
+                'required' => false,
+                'property_path' => 'validatorConfiguration[TweetValueValidator][authorList]',
+                'label' => 'field_definition.eztweet.authorList'
+            ]);
+    }
+}
+```
+
+Unfortunately, because in our case authorList validator has array type, we cannot leave it like that. We will need a way to transform data from array to comma-separated string and vice versa.
+To achieve this, first we need to implement a DataTransformer:
+
+```php
+// Form/StringToArrayTransformer.php
+
+namespace EzSystems\TweetFieldTypeBundle\Form;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * DataTransformer that transforms array into comma-separated string and vice versa
+ */
+class StringToArrayTransformer implements DataTransformerInterface
+{
+    public function transform($array)
+    {
+        if ($array === null) {
+            return '';
+        }
+
+        return implode(',', $array);
+    }
+
+    public function reverseTransform($string)
+    {
+        if (empty($string)) {
+            return [];
+        }
+
+        $array = explode(',', $string);
+
+        return array_map('trim', $array);
+    }
+}
+```
+
+Then, we can use this DataTransformer in our FormMapper like this:
+```php
+// eZ/Publish/FieldType/Tweet/FormMapper.php
+
+namespace EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet;
+
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface;
+use EzSystems\TweetFieldTypeBundle\Form\StringToArrayTransformer;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormInterface;
+
+class FormMapper implements FieldDefinitionFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                // Creating from FormBuilder as we need to add a DataTransformer.
+                $fieldDefinitionForm->getConfig()->getFormFactory()->createBuilder()
+                    ->create('authorList', TextType::class, [
+                        'required' => false,
+                        'property_path' => 'validatorConfiguration[TweetValueValidator][authorList]',
+                        'label' => 'field_definition.eztweet.authorList'
+                    ])
+                    ->addModelTransformer(new StringToArrayTransformer())
+                    // Deactivate auto-initialize as we're not on the root form.
+                    ->setAutoInitialize(false)->getForm()
+            );
+    }
+}
+```
+
+Next thing is to register the FormMapper as a service, so the system would know to use it. To achieve this, let's add the following lines to `fieldtypes.yml`:
```

Comment:
Add a link to the Services documentation page, here, please.
https://doc.ezplatform.com/en/1.11/guide/service_container/